src/Poisson2DSparseMatrix.cpp

```diff
@@ -170,7 +343,7 @@ namespace blitzdg{
 
                 real_type gtau = 100*100*2*(N+1)*(N+1)*hinv; // set penalty scaling
```

Comment:
Any significance to the factor `2*100*100`?

Reply:
It's just some kind of big-O / upper-bound factor. I think Warburton's book had 2*100 here, I found things were more robust with an extra factor of 100.... penalty methods are bit sketchy this way, and they've been criticized for thingslike this.